proc-macros/src/rpc_macro.rs

```diff
@@ -241,3 +247,14 @@ fn has_attr(attrs: &[Attribute], ident: &str) -> bool {
 	}
 	false
 }
+
+fn build_unsubscribe_method(existing_method: &str) -> String {
+	let method = existing_method.trim();
+	let mut new_method = String::from("unsubscribe");
+	if method.starts_with("subscribe") {
+		new_method.extend(method.chars().skip(9));
```

Comment:
Just for clarification, you are just appending anything after "subscribe" to the "unsubscribe" new method, and this method name originates from the subscriptions attributes, which can technically be any string?

Reply:
in a nutshell if the existing string is named `subscribeTarik` the unsubscribe method is named `unsubscribeTarik` hence the `skip(9)`

otherwise only `unsubcribe` is pushed at the start of the `subscribe method` for example `someSub` -> `unsubscribesomeSub`

It can be any string but if a duplicate string is used then it will end-up in compile-time error

Reply:
Cool, makes sense!